docs/source/conf.py

```diff
@@ -162,7 +162,7 @@ def patched_parse(self):
 # built documents.
 #
 # The short X.Y version.
-with open(os.path.join("../../scico", "__init__.py")) as f:
+with open(os.path.join(confpath, "../../scico", "__init__.py")) as f:
     version = parse(next(filter(lambda line: line.startswith("__version__"), f))).body[0].value.s
```

Comment:
Why was this change necessary?

Also, although not introduced in this PR, one could argue that it's a bit pointless to separate the final two path components, to be joined by `os.path.join`, when one of those components (`../../scico`) had non-os-independent path separation. Perhaps worth fixing this now, making it `os.path.join(confpath, "..", "..", "scico", "__init__.py")`?

Reply:
I was getting an error in either the doc tests or the doc build without it, but I don't recall which.

Separators fix.